pex/environment.py

```diff
@@ -220,7 +225,8 @@ def can_add(self, dist):
       return True
 
     try:
-      wheel_name_, wheel_raw_version_, wheel_tags = filename.split('-', 2)
+      whl = wheel.Wheel(filename)
```

Comment:
Unfortunately, this reads in and hashes the full wheel file: https://github.com/pypa/wheel/blob/a51977075740fda01b2c0e983a79bfe753567219/src/wheel/wheelfile.py#L38-L79
We definitely don't want to do this for many wheels used in the ML world for example since those wheels are routinely massive. Although you could jump through some hoops here and set the mode to 'w' but never actually write, etc - that's fragile and getting a bit ugly.

How do you feel about just copying the proper regex from that code and using it here? Its a bit unsatisfying, but we pick up no new runtime dependency and we do less work. Its nt fragile since this is all spec'd in [PEP 427](https://www.python.org/dev/peps/pep-0427/#file-name-convention).

Reply:
Yeah, we have lots of build code which uses `Wheel(filename)`, so I thought it was an easy solution here, but as I learned that's **not** the simple path. Even a regex is pretty unnecessary when all we need are the trailing 3 parts of the wheel filename.

I'm simplified the solution to (essentially):
```
wheel_tags = '-'.join(filename.split('-')[-3])
```